lib/msal-browser/src/cache/DatabaseStorage.ts

```diff
@@ -234,21 +234,23 @@ export class DatabaseStorage<T> implements IAsyncStorage<T> {
         if (this.db && this.dbOpen) {
             this.closeConnection();
         }
-        
-        // @ts-ignore
-        const existingDatabases = await window.indexedDB.databases();
-        const database = existingDatabases.find((database: IDBDatabaseInfo) => database.name === DB_NAME );
-
-        // If database exists, delete it
-        if (database) {
-            return new Promise<boolean>((resolve: Function, reject: Function) => {
-                const deleteDbRequest = window.indexedDB.deleteDatabase(DB_NAME);
-                deleteDbRequest.addEventListener("success", () => resolve(true));
-                deleteDbRequest.addEventListener("error", () => reject(false));
-            });
+
+        try {
+            // @ts-ignore
+            const existingDatabases = await window.indexedDB.databases();
+            const database = existingDatabases.find((database: IDBDatabaseInfo) => database.name === DB_NAME );
+            return (database) ? await this.deleteCurrentDatabase() : true;
```

Comment:
```suggestion
            if (database) {
                return await this.deleteCurrentDatabase();
            }
            return true;
```

Reply:
Nitpick, a little simpler to read.

Reply:
And a comment explaining why true is being returned as a fallback would be good.